Automattic/jetpack
# Social: Fix "No media" notice for Instagram when SIG is ON

<details>

<summary>When SIG is enabled, media notice is still shown, and Instagram connection is disabled.</summary>

![Image](https://github.com/user-attachments/assets/f0cfa949-38aa-4ea2-a320-94f0d69c425e)

</details>



## Proposed changes:

* Remove media required notice for Instagram when Social Image Generator is ON

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to post editor
* Remove any attached media
* Turn ON Social Image Generator
* Confirm that Instagram required media notice is not shown
* Confirm that the Instagram connection is not disabled and can toggled
